### ISSUE TITLE
Search failed: "^|=+=|$" trying to (mastodon-toot--send)

### ISSUE THREAD

**Issue by saper** [martianhi/mastodon.el#103]
This is mastodon-20170428.1657 from melpa on a pretty fresh emacs 25 with evil mode and emojify installed (nothing else).

When trying to reply to https://mastodon.xyz/users/ztbrown/updates/134897 (boosten by @jdenen to my home timeline) I write this in the window

````
@johnson @ztbrown@mastodon.xyz and this is a reply from #mastodon.el :)
````

After this `C-c c-c` or running `mastodon-toot--send` fails with

````
Search failed: "^|=+=|$" 
````

in the minibuffer

Important note: There are no docs displayed in the toot window. No error, either.

**Comment by saper** [comment]
It seems that we are using `caadr`, `remove-if-not` and `string-join` which are not elisp

**Comment by jdenen** [comment]
Fixed by #103, which was rebased and pushed straight to develop.
